src/nvim/buffer.c

```diff
@@ -1335,11 +1336,12 @@ void do_autochdir(void)
 static int top_file_num = 1;            /* highest file number */
 
 buf_T *
-buflist_new (
-    char_u *ffname,            /* full path of fname or relative */
-    char_u *sfname,            /* short fname or NULL */
-    linenr_T lnum,                  /* preferred cursor line */
-    int flags                      /* BLN_ defines */
+buflist_new(
+    char_u *ffname,            // full path of fname or relative
+    char_u *sfname,            // short fname or NULL
+    linenr_T lnum,             // preferred cursor line
+    int flags,                 // BLN_ defines
+    handle_T bufnr
```

Comment:
instead of this, was there discussion about maybe using `BLN_DUMMY`? (still investigating this myself)